### PR TITLE
Update to be compatible with latest holochain

### PR DESCRIPTION
**Pull request by timotree3**
Supercedes #59

### CODE REVIEW

**Comment by timotree3** [comment]
Testing locally

**Comment by timotree3** [comment]
actually just gonna test it deployed

**Comment by timotree3** [comment]
works with new holochain

File: src/store/holochain.js [hunk 1]
```diff
@@ -60,7 +60,13 @@ const initializeClientHolo = async (commit, dispatch, state) => {
...
   const appInfo = await holoClient.appInfo()
   const [cell] = appInfo.cell_data
-  const [cellId, dnaAlias] = cell
+  let cellId, dnaAlias
+  if (Array.isArray(cell)) {
+    [cellId, dnaAlias] = cell
+  } else {
+    cellId = cell.cell_id
+    dnaAlias = cell.cell_nick
+  }
```
Comment (robbiecarlton) [inline src/store/holochain.js:69]: not blocking, but this would probably be a good thing to put in utils

**Review by robbiecarlton — APPROVED** [review]

**Pull request merged**